.travis.yml

```diff
@@ -7,11 +7,6 @@ os: osx
 osx_image: xcode6.4
 
 env:
-  matrix:
-    
-    - CONDA_PY=27
-    - CONDA_PY=35
-    - CONDA_PY=36
```

Comment:
Hmm...do we know why these disappeared?

Reply:
Good catch, I didn't check these other changes, I've just continued from where the bot PR stopped.

Reply:
Yeah, have seen these issues in the past with re-rendering. Though that is not a common occurrence with the re-rendering bot or manual re-renderings any more. Not sure why the autoupdate bot is running into this issue, but have seen it in a couple of cases. Normally our bot or manually re-rendering has fixed it afterwards. Could be anything from a connectivity issue or suppressed error.

Letting you know for awareness, @CJ-Wright. Probably not worth worrying about to much, but if there is an issue to track this that would be good for now. If you have any ideas, they would of course be welcome, but just being aware is probably good for now given cb3 and cs3 are rolling out too.

Reply:
it was because `boost-cpp` PR was not merged at the time.

Reply:
Good point. Thanks for mentioning that.